.vib/acmesolver/vib-publish.json

```diff
@@ -78,14 +78,14 @@
           "action_id": "container-image-publish",
           "params": {
             "metadata": {
-              "tags": {VIB_ENV_ROLLING_TAGS}
+              "tags": "{VIB_ENV_ROLLING_TAGS}"
```

Comment:
Have you tested it? `tags` requires an array and you are passing a string

Reply:
You are right, this is failing the DSL schema validation, as it expects an array. I don't see an easy solution, we would need to add or parse the `tags` field directly in the CD pipeline to avoid this issue.

I will revert this change back and make do with the parsing error, IMO it is not critical enough to obscure part of the DSL by adding logic elsewhere. wdyt @fmulero, @joancafom?

Reply:
LGTM. We can survive at the moment. Also we have plans to change these files in order to reuse as much as possible.